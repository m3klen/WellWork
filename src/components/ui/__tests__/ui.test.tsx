/**
 * @jest-environment jsdom
 */

import '@testing-library/jest-dom'
import { render } from '@testing-library/react'

import UI from '../'

describe('components:ui', () => {
  it('matches snapshot with default test props', async () => {
    const { container } = render(<UI />)

    expect(container).toMatchSnapshot()
  })

  it('uses the provided class name', async () => {
    const className = 'test-class'
    const { container } = render(<UI className={className} />)

    expect(container).toMatchSnapshot()
  })
})
