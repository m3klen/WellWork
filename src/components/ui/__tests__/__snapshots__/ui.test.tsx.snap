// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components:ui matches snapshot with default test props 1`] = `
<div>
  <div
    class="component-ui"
  >
    <div
      class="page-home"
    >
      <div
        class="page-home-content-wrapper"
      >
        <h2>
          React TypeScript Web App Template
        </h2>
      </div>
    </div>
  </div>
</div>
`;

exports[`components:ui uses the provided class name 1`] = `
<div>
  <div
    class="component-ui test-class"
  >
    <div
      class="page-home"
    >
      <div
        class="page-home-content-wrapper"
      >
        <h2>
          React TypeScript Web App Template
        </h2>
      </div>
    </div>
  </div>
</div>
`;
